models/rental.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const Rental =  mongoose.model('Rental', new mongoose.Schema({
+    
+  userId:{
+        type: mongoose.Types.ObjectId, 
+        ref: 'User'
+
+    },
+
+    appartmentId:{
+        type: mongoose.Types.ObjectId, 
+        ref: 'Rental'
```

Comment:
Shouldn't ref be `Appartment` instead?

Reply:
yes, sir it should be Apartment mistake!